Permanent/ViewControllers/Profile/ProfilePageViewController.swift

```diff
@@ -0,0 +1,301 @@
+//
+//  ProfileViewController.swift
+//  Permanent
+//
+//  Created by Lucian Cerbu on 09.11.2021.
+//
+
+import UIKit
+
+class ProfilePageViewController: BaseViewController<ProfilePageViewModel> {
```

Comment:
can we rename these to PublicProfilePageViewController and PublicProfilePageViewModel? I'm thinking we can avoid some confusion down the line if we clearly specify these are related to Public Archives